autoleagueplay/match_exercise.py

```diff
@@ -79,6 +79,7 @@ def on_tick(self, tick: TrainingTickPacket) -> Optional[Grade]:
             time.sleep(1)  # Give time for replay_monitor to register replay and for RL to load main menu
             if self.replay_monitor.replay_id or self.replay_monitor.replay_preference == ReplayPreference.IGNORE_REPLAY:
                 self.replay_monitor.stop_monitoring()
+                self.replay_monitor.anonymize_replay()
```

Comment:
The ReplayMonitor does not have this function, `anonymize_replay()`. There is a similarly named function `replays.py` file though. You also try to make this call in line 92 and 97.